### PR TITLE
Module 1 Capstone Project for code review.

### PR DESCRIPTION
**Pull request by demix007**
# In the process of building this capstone project, I did the following:
- Set up the Linters and ensured that there are no linter errors. 
- Used the correct GitHub flow.
- Used the recommended design guidelines including colors, typography, and layout.
- Built the first two pages: The home page and the about page.
- Ensured that each page has versions for 2 different sizes: the mobile (up to 768px wide) and the desktop(768px or wider) versions.
- Created interactions on the site (Links and mobile menu).
- Created the section "Featured Speakers" dynamically in JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,38 @@
...
-
+- HTML, CSS & JAVASCRIPT.
 
+## Live Demo    
 
-## Authors
+[Live Demo Link](https://demix007.github.io/Capstone-Project-Style-Conference/)
 
-👤 **Author1**
+[Live Demo Link](https://www.loom.com/share/afa22ca17b604af38c8cc5b72b1f09f9)
```
Comment (fobadara) [inline README.md]: - [x] [REQUIRED] There is something wrong with your video. It is not playing despite trying it on about 3 browsers.
- [OPTIONAL] I think you should have named this something like `video` or `video presentation`

File: README.md [hunk 2]
```diff
@@ -1,62 +1,38 @@
...
+> This project is a final capstone project for HTML/CSS & JavaScript. In order to meet the project requirements, I had to implement the following: 
+- Set up the Linters and ensured that there are no linter errors. 
+- Used the correct GitHub flow.
+- Used the recommended design guidelines including colors, typography, and layout.
+- Built the first two pages: The home page and the about page.
+- Ensured that each page has versions for 2 different sizes: the mobile (up to 768px wide) and the desktop(768px or wider) versions.
+- Created interactions on the site (Links and mobile menu).
+- Created the section "Featured Speakers" dynamically in JavaScript.    
 
 ## Built With
```
Comment (fobadara) [inline README.md:8]: - [x] [REQUIRED] Kindly give us details on how we can use your website.

File: README.md [hunk 3]
```diff
@@ -1,62 +1,38 @@
...
+# HTML/CSS & JavaScript capstone project - Conference Page
 
+> This project is a final capstone project for HTML/CSS & JavaScript. In order to meet the project requirements, I had to implement the following: 
+- Set up the Linters and ensured that there are no linter errors. 
+- Used the correct GitHub flow.
+- Used the recommended design guidelines including colors, typography, and layout.
+- Built the first two pages: The home page and the about page.
+- Ensured that each page has versions for 2 different sizes: the mobile (up to 768px wide) and the desktop(768px or wider) versions.
+- Created interactions on the site (Links and mobile menu).
+- Created the section "Featured Speakers" dynamically in JavaScript.
```
Comment (fobadara) [inline README.md]: [OPTIONAL] I think this sounds. more like a PR summary than an intro.  This space is for you to tell us about what your site does and what makes it special.

File: README.md [hunk 4]
```diff
@@ -1,62 +1,38 @@
...
 
-👤 **Author2**
+👤 **Ayodeji Abidemi**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/demix007)
+- Twitter: [@twitterhandle](https://twitter.com/dat_dope_demix)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ayodeji-abidemi-b76935218/)
```
Comment (fobadara) [inline README.md:34]: - [x] [REQUIRED] Kindly credit Cindy (the owner of the design on Behance) in your acknowledgements section of your README as instructed. And don't forget to include a link.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+              </a>
+            </div>
+        
+            <i class="fas fa-bars fa-3x text-white cursor-pointer" id="openMenu" aria-hidden="true" ></i>
+    
+            <nav class="menu-nav" id="mobile-nav">
+              <i class="fa fa-times fa-3x text-white" aria-hidden="true" id="closeMenu"></i>
+              <ul class="menu-nav-items">
+                <li class="page-list lato-font home">
+                  <a href="index.html">Home</a>
```
Comment (fobadara) [inline about.html:57]: - [x] [REQUIRED] For some reason your `home` button is not showing on the about page. Kindly fix this.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+                  <a href="#">Reach Out</a>
+                </li>
+                <li class="page-list lato-font">
+                  <a href="#">Gallery</a>
+                </li>
+              </ul>
+              <div class="product">
+                <a class="font-lato" href="index.html">Community Day</a>
+              </div>
+            </nav>
```
Comment (fobadara) [inline about.html:78]: - [x] [REQUIRED] Kindly fix your hamburger button for the about page. It doesn't display the popup navigation when it is clicked.
- [x] [REQUIRED] Kindly make your navigation pop up transparent

File: index.html [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+                <i class="fas fa-bars fa-3x white-text" aria-hidden="true" id="openMenu"></i>
+
+                <nav class="menu-nav" id="mobile-nav">
+                    <i class="fas fa-times fa-3x white-text" aria-hidden="true" id="closeMenu"></i>
+                    <ul class="menu-nav-items">
+                        <li class="page-list lato-font home-link">
+                            <a href="index.html" class="active-tab">Home</a>
+                        </li>
+                        <li class="page-list lato-font">
+                            <a href="about.html" class="active-tab">About</a>
```
Comment (fobadara) [inline index.html]: - [OPTIONAL] I think you should make only the current page orange colour

File: index.html [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+                    </p>
+                </div>
+                <div class="event-info">
+                    <h3 class="date">2022.09.25 (Fri) ~ 27(Sun)</h3>
+                    <p class="venue lato-font">
+                        @ LightHouse Christian Outreach Center
+                    </p>
+                </div>
+            </div>
+        </div>
```
Comment (fobadara) [inline index.html:103]: - [x] [REQUIRED] Kindly make sure the corrections pointed out matches Cindy's design. One of the purposes of this task is to teach to follow client requirements.

| Your Design|
|-------------|
|image|

File: index.html [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+                <div class="event4 event">
+                    <img src="img/game4.png" alt="event-icon" class="event-img event-icon">
+                    <h2 class="lato-font">Games</h2>
+                    <p class="lato-font">
+                        Games including football, tennis, basketball, and board games.
+                    </p>
+                </div>
+
+                <div class="event5 event">
+                    <img src="img/game5.png" alt="event-icon" class="event-img event-icon">
```
Comment (fobadara) [inline index.html:145]: - [x] [REQUIRED] Kindly take note of the differences between your design and. Cindy's in the image below:

|Your Design|
|-------------|
|image|

File: index.html [hunk 10]
```diff
@@ -0,0 +1,209 @@
...
+            </div>
+        </div>
+    </section>
+
+    <!--Featured Speakers Section-->
+    <div class="featured-speakers-heading">
+        <h2 class="lato-font">Featured Speakers</h2>
+        <hr>
+    </div>
+
```
Comment (fobadara) [inline index.html:169]: - [x] [REQUIRED]  kindly check the image below:

|Your Design|
|----------|
|image|

- [x] [REQUIRED] Kindly reduce the width and  make the speakers in the centre for the mobile version

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @demix007, 

Good job so far!
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

You can also consider:

- [OPTIONAL] Good job  👍🏽  following some of the best practices for writing commit messages but I think your commit message for a project as large as this should be more.

- [OPTIONAL] I don't think you need to include `-branch` in the title of your feature branch. we also know you include features in this branch but I think we can use a more specific name for projects like this. Perhaps `full_project`. You can check out how to change a branch name  [here](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

- [OPTIONAL] I also noticed that the name of your repo doesn't tell us much at first glance. You are going to have a lot of capstone projects. And imagine getting a job and naming the repo job. I think something like lighthouse-2022 might have been better. Kindly treat these projects like you are working a real job.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 11]
```diff
@@ -1,72 +1,50 @@
...
-- Technologies used
+- HTML, CSS & JAVASCRIPT.
 
-## Live Demo (if available)
+## Live Demo    
 
-[Live Demo Link](https://livedemo.com)
+[Live Site Demo](https://demix007.github.io/Capstone-Project-Style-Conference/)
 
+[Project Description Video](https://www.loom.com/share/afa22ca17b604af38c8cc5b72b1f09f9)
```
Comment (danielufeli) [inline README.md]: - [x] Nice working on your video presentation, like the previous reviewer mentioned the video is not loading, kindly fix this.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @demix007 , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Design ✔️

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success

## Hey @demix007 :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following the GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title
✔️ Following the Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (demix007) [reply README.md]: ## Hello @fobadara 👋🏼,

Thank you for reviewing, the changes which you required have been implemented.

File: README.md (hunk 3 above)
Comment (demix007) [reply README.md]: I have also implemented this change.

**Comment by demix007** [comment]
## Hello @HAMMAS-SALEEM 👋🏼 ,

Thank you for taking out time to review and approve this PR. You are amazing 💯

**Pull request merged**